package.json

```diff
@@ -45,6 +45,7 @@
     "@types/react": "16.9.11",
     "@types/react-color": "3.0.1",
     "@types/react-dom": "16.9.4",
+    "@types/styled-components": "4.1.8",
```

Comment:
The latest version of `@types/styled-components` is `4.4.0`.

However, this project needs to keep a lower version temporarily due to type definition conflicts. See [pull request](https://github.com/microsoft/types-publisher/pull/655#issue-317569866) for more information. 

To recreate the conflicts, follow these steps:

1) Upgrade `@types/styled-components` to `4.4.0`.
2) Run `yarn tsc`.
3) TypeScript shows compile errors due to conflicting types

Reply:
Take a look at https://github.com/zendeskgarden/react-components/blob/master/.yarnclean

Also take a look at https://github.com/zendeskgarden/react-components/blob/master/package.json#L131-L133 if you run into `@types/react` issues 

We are able to remove this invalid dependency in `react-components` successfully

Reply:
Awesome, that resolves the type definition conflicts and we can use the latest (v4.4.0) of `@types/styled-components`.